bin/deepstate/core/fuzz.py

```diff
@@ -210,26 +192,107 @@ def parse_args(cls) -> Optional[argparse.Namespace]:
       "--fuzzer_help", action="store_true",
       help="Show fuzzer command line interface's help options.")
 
+    parser.add_argument(
+      "--home_path", type=str,
+      help="Path to fuzzer home directory. Will search executables there and in PATH.")
+
     # finalize building up parser by passing to superclass, and instantiate object attributes
     # the base `parse_args` sets state with _ARGS, so we do need to return a namespace
     cls.parser = parser
     super(FuzzerFrontend, cls).parse_args()
 
+    # parse fuzzer_args
+    _args: Dict[str, Any] = vars(cls._ARGS)
+
+    fuzzer_args_parsed: List[Tuple[str, Optional[str]]] = []
+    for arg in _args['fuzzer_args']:
```

Comment:
This logic is duplicated; maybe we should move it to a function and use it for fuzzer args here and target args in the other file?